src/js/views/main/welcome.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import styled from 'styled-components';
+
+const Container = styled.div`
+   z-index: 1;
+   display: flex;
+   align-items: center;
+   flex: 1;
+   max-width: 33em;
+   max-height: ${props => props.isReady ? '30%' : '55%'};
+   transition: all 0.3s ease;
+`;
+
+const Subtext = styled.h2`
+   font-weight: 300;
+   font-size: 42px;
+   color: rgb(70, 70, 70);
+   margin: 0;
+   animation: slideUp 0.5s;
+`;
+
+const Emoji = styled.h2`
+   width: 60px;
+   height: 60px;
+   font-size: 50px;
+   margin: 0 24px;
+   animation: jiggle 1s cubic-bezier(0.68, -0.55, 0.265, 1.55);
+
+   @keyframes jiggle {
+      0% {
+         transform: rotate(-30deg) scale(0.5);
+         opacity: 0;
+      }
+   }
+`;
+
+const Text = styled.h3`
+   margin: 0 auto;
+   color: #2c4358;
+   font-size: 23px;
+   font-weight: 500;
+   animation: slideUp 0.5s;
+
+   @keyframes slideUp {
+      0% {
+         transform: translateX(-10%);
+         opacity: 0;
+      }
+   }
+`;
+
+const Welcome = ({ isReady, uploading }) => {
+   return (
+      <Container isReady={isReady}>
+         <Emoji>{!isReady ? "👋" : uploading ? "🧐" : "🎉"}</Emoji>
+         <div>
+            {isReady && !uploading && (
+               <React.Fragment>
+                  <Subtext>Ready to Upload</Subtext>
+                  <Text>Feel free to add more files if you'd like.</Text>
```

Comment:
We should put this in a template literal (backticks)

Reply:
Agreed